### PR TITLE
NODE-1938: IssueTransaction gRPC tests and PBTransactions fix

### PR DESCRIPTION
**Pull request by finnoio**
1. Tests for gRPC IssueTransaction
2. Fix script validation error in PBTransactions.createVanilla()

### CODE REVIEW

File: node-it/src/test/scala/com/wavesplatform/it/sync/grpc/IssueTransactionGrpcSuite.scala [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import com.wavesplatform.lang.ValidationError
+import com.wavesplatform.transaction.TxValidationError.{InvalidName, NonPositiveAmount, TooBigArray}
+import io.grpc.Status.Code
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class IssueTransactionGrpcSuite extends GrpcBaseTransactionSuite with NTPTime with TableDrivenPropertyChecks {
+
+  val (issuer,issuerAddress) = (firstAcc,firstAddress)
+
+  test("asset issue changes issuer's asset balance") {
```
Comment (DrBlast) [inline node-it/src/test/scala/com/wavesplatform/it/sync/grpc/IssueTransactionGrpcSuite.scala]: add issue nft asset and with decimals 8
also add tests for boundary conditions for asset name

File: node-it/src/test/scala/com/wavesplatform/it/sync/grpc/IssueTransactionGrpcSuite.scala [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+  test("Able to issue NFT asset") {
+    for (v <- supportedVersions) {
+      val assetName = Random.alphanumeric.filter(_.isLetter).take(IssueTransaction.MinAssetNameLength + 1).mkString
+      val assetDescription = ByteString.copyFrom("nft asset".getBytes(StandardCharsets.UTF_8))
+      val issuerBalance = sender.grpc.wavesBalance(issuerAddress).available
+      val issuerEffBalance = sender.grpc.wavesBalance(issuerAddress).effective
+      val (nftQuantity, nftDecimals, nftReissuable) = (1, 1, false)
+
+      val issuedAssetTx = sender.grpc.broadcastIssue(issuer, assetName, nftQuantity, nftDecimals, nftReissuable, issueFee, assetDescription, version = v, script = scriptText(v), waitForTx = true)
```
Comment (DrBlast) [inline node-it/src/test/scala/com/wavesplatform/it/sync/grpc/IssueTransactionGrpcSuite.scala]: nft fee is minimal tx fee, not issue fee

File: node-it/src/test/scala/com/wavesplatform/it/sync/grpc/IssueTransactionGrpcSuite.scala [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    }
+  }
+
+  test("Able to issue NFT asset") {
+    for (v <- supportedVersions) {
+      val assetName = Random.alphanumeric.filter(_.isLetter).take(IssueTransaction.MinAssetNameLength + 1).mkString
+      val assetDescription = ByteString.copyFrom("nft asset".getBytes(StandardCharsets.UTF_8))
+      val issuerBalance = sender.grpc.wavesBalance(issuerAddress).available
+      val issuerEffBalance = sender.grpc.wavesBalance(issuerAddress).effective
+      val (nftQuantity, nftDecimals, nftReissuable) = (1, 1, false)
```
Comment (DrBlast) [inline node-it/src/test/scala/com/wavesplatform/it/sync/grpc/IssueTransactionGrpcSuite.scala]: nft-decimals = 0
